common.php

```diff
@@ -1,9 +1,64 @@
 <?php
 
 require_once 'config/main.php';
-
+require_once 'src/repository/ClientRepository.php';
+require_once 'src/repository/CommentRepository.php';
 $connection = new PDO($dsn, $username, $password, $options);
 
+$clientRepository = new ClientRepository($connection);
+$commentRepository = new CommentRepository($connection);
+
 function escape($html) {
     return htmlspecialchars($html, ENT_QUOTES | ENT_SUBSTITUTE,"UTF-8");
 }
+
+
+function template($file, $args){
```

Comment:
1. Add type of data which acceptable by this function.
2. Add type of data which will be returned.
3. Check implementation that in any logical case we will return our data type which will be expected.